### DISCUSSION TITLE
Hey, my guy, link discord.

### DISCUSSION THREAD

**Discussion by XenoTheStrange** [pokepetter/ursina#711] [Ideas]
You should attach a discord server to this project so the community can more easily collaborate.

I was legit just like "is this engine dead?" But I checked the issues and updates in the code and it's alive af, so

You should make a discord server for this. People can collab. Much ":D"
Pls.

**Comment by XenoTheStrange** [comment]
I wanna vibe with the other bearfolk. 
The ursine sorts
Ya' know.

**Comment by pokepetter** [comment]
We have a Discord server. Click the Discord icon on the ursina web page.

**Reply by XenoTheStrange** [reply]
Ok, cool, thanks.
You could link it in the Readme for this repo, though, so more people see it 👀

**Reply by pokepetter** [reply]
That's a good idea, thanks.
